src/main/java/com/fundynamic/d2tm/game/map/MapEntity.java

```diff
@@ -0,0 +1,30 @@
+package com.fundynamic.d2tm.game.map;
+
+import com.fundynamic.d2tm.game.behaviors.Selectable;
+import com.fundynamic.d2tm.math.Vector2D;
+import org.newdawn.slick.Graphics;
+import org.newdawn.slick.SpriteSheet;
+
+/**
+ * An entity bound to the map
+ */
+public abstract class MapEntity {
```

Comment:
I agree. We should think of naming here as well. I prefixed Map because it has a relation with it. Although Units have that as well. Perhaps we should store it with entities and try to not separate them?

On Sat, Feb 21, 2015 at 12:14 AM, Arjen van der Ende
<notifications@github.com> wrote:

>> @@ -0,0 +1,30 @@
>> +package com.fundynamic.d2tm.game.map;
>> +
>> +import com.fundynamic.d2tm.game.behaviors.Selectable;
>> +import com.fundynamic.d2tm.math.Vector2D;
>> +import org.newdawn.slick.Graphics;
>> +import org.newdawn.slick.SpriteSheet;
>> +
>> +/**
>> + * An entity bound to the map
>> + */
>> +public abstract class MapEntity {
> I feel the `Map` prefix is used inconsistently. We have it here, but it's `Unit` and not `MapUnit` or `MapStructure`. Maybe rename it to `Entity` or are we expecting other kinds of entities?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/stefanhendriks/dune2themaker4j/pull/42/files#r25111579